go.mod

```diff
@@ -63,14 +63,13 @@ require (
 	cosmossdk.io/depinject v1.1.0 // indirect
 	filippo.io/edwards25519 v1.1.0 // indirect
 	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 // indirect
-	github.com/99designs/keyring v1.2.2 // indirect
-	github.com/DataDog/datadog-go v4.8.3+incompatible // indirect
+	github.com/99designs/keyring v1.2.1 // indirect
+	github.com/DataDog/datadog-go v3.2.0+incompatible // indirect
```

Comment:
[no change needed] odd that this was downgraded

Reply:
[052dd04](https://github.com/celestiaorg/celestia-app/pull/4589/commits/052dd04a0948aa53211b9e5282ddb0f98b6b4af4)